docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -264,6 +267,95 @@ Name | Type | Description
 * logs
 * traces
 
+## Choosing a load balancing strategy
+
+<!-- TODO: Mention gropubytrace processor when Flow supports it -->
+<!-- TODO: Should we run more than 1 LB instance for better resiliency and spreading out the load? -->
+
+There are various stateful Flow components which require specific load balancing strategies.
+<!-- TODO: "stateful" should be a link to the other doc which defines it -->
+
+### otelcol.processor.tail_sampling
+<!-- TODO: Add a picture of the architecture?  -->
+All spans for a given trace ID must go to the same tail sampling Agent.
+* This can be done by configuring `otelcol.exporter.loadbalancing` with `routing_key = "traceID"`.
+* If this is not done, the sampling decision may be incorrect. This is because the tail sampler must
+  have a full view of the trace when making a sampling decision.
+  * For example, a `rate_limiting` tail sampling strategy may incorrectly pass through 
+    more spans than expected if the spans for the same trace are spread out to more than one Agent instance.
+
+<!-- Make "rate limiting" a URL to the tail sampler doc -->
+
+### otelcol.connector.spanmetrics
+All spans for a given `service.name` must go to the same spanmetrics Agent.
+* This can be done by configuring `otelcol.exporter.loadbalancing` with `routing_key = "service"`.
+* If this is not done, metrics generated from spans might be incorrect.
+  * For example, if similar spans for the same `service.name` end up on different Agent instances,
+    the two Agents will have identical metic series for calculating span latency, errors, and number of requests.
+  * When both Agents attempt to remote write the metrics to a database such as Mimir,
+    the series with clash with each other.
+  * At best this will lead to an error in the Agent and a rejected remote write. 
+  * At worst, it could lead to an inaccurate data dure to overlapping samples for the metric series.
+
+However, there are ways to scale `otelcol.connector.spanmetrics` without the need for a load balancer:
+1. Each Agent could add an attribute such as `agent.id` in order to make its series unique.
+  * A `sum by` PromQL query could be used to aggregate the metrics from different Agents.
+  * An extra `agent.id` attribute has a downside that the metrics stored in the database will have extra cardinality.
+2. Spanmetrics could be generated in the backend database instead of the Agent.
+  * For example, span metrics can be [generated][tempo-spanmetrics] in Grafana Cloud by the Tempo traces database.
+
+[tempo-spanmetrics]: https://grafana.com/docs/tempo/latest/metrics-generator/span_metrics/
+
+### otelcol.connector.servicegraph
+Unfortunately, there is generally no reliable way to scale `otelcol.connector.servicegraph` over multiple Agent instance.
+<!-- Or is there? Can we fix the issue below using a PromQL query? There may still be higher than normal cardinality though? -->
```

Comment:
On Mimir we can do read queries which aggregate data from multiple Agent instances. However, the data will still have high cardinality because Mimir has to store the "agent_id" label. I don't think there is a way to reduce it unless people use adaptive metrics somehow.